cheng-k/2025-Summer.md

```diff
@@ -6,32 +6,83 @@
 
 #### 🌹
 
-* 
+* I now have a good understanding about BIDS, goals it strives to address, and ideas it stands for. I am looking forward to continuing to work on these initiatives!
 
 #### 🌱
 
-* 
+* I have learned a lot about open source in the past few days. I am amazed at the quality of work put in by passionate volunteers. I never realized that there needs to be documentation on how to write documentation. So many products use open source tools such as Python, and it's all thanks to the collective work of thousands (millions?) of people who together want to create a useful and accessible tool. Managing a repository and its growing community is no small feat behind the scenes.
```

Comment:
> I never realized that there needs to be documentation on how to write documentation.

OMG so true!!

I love this reflection so much - thank you for thinking about open source so deeply ✨ 

Please can you put these sentences on new lines (as in my first review comment) - thank you!